Mu2eKinKal/inc/KKFitSettings.hh

```diff
@@ -0,0 +1,62 @@
+#ifndef Mu2eKinKal_KKFitSettings_hh
+#define Mu2eKinKal_KKFitSettings_hh
+//
+// Struct for configuring the Mu2e KinKal fit 
+//
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
+#include "fhiclcpp/types/Table.h"
+#include "fhiclcpp/types/Tuple.h"
+#include "KinKal/Fit/Config.hh"
+#include "DataProducts/inc/PDGCode.hh"
+#include "RecoDataProducts/inc/TrkFitDirection.hh"
+namespace mu2e {
+  namespace Mu2eKinKal{
+    using Name    = fhicl::Name;
+    using Comment = fhicl::Comment;
+
+    // struct for defining the KinKal Config object
+    struct KKConfig {
+      fhicl::Atom<int> maxniter { Name("MaxNIter"), Comment("Maximum number of algebraic iteration steps in each fit meta-iteration"), 10 };
```

Comment:
are all these default necessary? or they could live in the corresponding prolog file?